pkg/reconciler/v1alpha1/route/resources/virtual_service.go

```diff
@@ -36,6 +36,7 @@ const (
 	PortName   = "http"
 
 	DefaultRouteTimeout = "60s"
```

Comment:
Please remove this?

I think it may be referred to by tests, but we should replace that with fmt.Sprintf("%ds", DefaultRouteTimeoutSeconds)  to be consistent.  Probably could extract that into a test constant if you want brevity.

Reply:
Done. I replaced all of DefaultRouteTimeout with  fmt.Sprintf("%ds", DefaultRouteTimeoutSeconds)